src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGeneration.Telemetry.cs

```diff
@@ -107,7 +107,6 @@ private void TrackStartGeneration(XamlFileDefinition[] files)
 					_telemetry.TrackEvent(
 						"generate-xaml",
 						new[] {
-							("IsWasm", _isWasm.ToString()),
```

Comment:
Note: This piece of info will be available in UnoRuntime value. So we haven't actually lost that info.

Reply:
Let's keep it for now, even if duplicated.